docs/design/DB/dokuha-roadmap.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE Users (
+	user_id SERIAL PRIMARY KEY,
+	username VARCHAR(255) NOT NULL,
+	password VARCHAR(255) NOT NULL
+);
+
+
+
+CREATE TABLE Contents (
+	content_id SERIAL PRIMARY KEY,
+	users_id INT NOT NULL,
```

Comment:
users_id -> user_id でいいと思う。
RDB的に、実際にこの FK でつながる先は　ユーザー"達" じゃなくて ユーザー一人に対してたくさんの本が紐づくので！

```suggestion
	user_id INT NOT NULL,
```